ark/segmentation/marker_quantification_test.py

```diff
@@ -56,6 +56,17 @@ def test_compute_marker_counts_base():
     assert np.array_equal(segmentation_output.loc['whole_cell', :, settings.CELL_SIZE],
                           segmentation_output.loc['whole_cell', :, 'area'])
 
+    # test different extraction selection
+    center_extraction = \
+        marker_quantification.compute_marker_counts(input_images=input_images,
```

Comment:
I think it would be more useful to test the top level function, `generate_cell_table`, to make sure all the appropriate kwargs are getting propagated through